### ISSUE TITLE
NTR, pulmo: Platypnea-Orthodeoxia syndrome

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6479]
**Preferred term label:**
Platypnea-Orthodeoxia syndrome

**Synonyms**
Abbrevation POS

**Definition (free text, please give PubMed ID)**
Platypnea-Orthodeoxia syndrome (POS) is a rare clinical entity characterized by dyspnea and arterial desaturation while changing from a recumbent to an upright position

Comment: The hypoxia in POS has been attributed to the mixing of the deoxygenated venous blood with the oxygenated arterial blood via a shunt. This can be due to intracardiac or extracardiac abnormalities (including intra-pulmonary arteriovenous malformations and lung parenchymal diseases) or miscellaneous etiologies.

**Parent term (use hpo.jax.org/app)**
Functional respiratory abnormality HP:0002795

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by GrieseAG** [comment]
Possible further terms: 
Platypnea: Subterm of Dyspnea HP:0002094> A attack of dyspnea that occurs while the affected person when changing from a recumbent to an upright position 
Orthodeoxia: Subterm of Hypoxemia HP:0012418 > Low level of blood oxygen when changing from a recumbent to an upright position

**Comment by pnrobinson** [comment]
added these terms. I think we should not add Platypnea-Orthodeoxia syndrome because it should be coded by the two individual terms (bundling)
